common/src/main/java/com/wynntils/wc/custom/item/properties/PowderTierProperty.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.custom.item.properties;
+
+import com.wynntils.features.user.ItemHighlightFeature;
+import com.wynntils.features.user.ItemTextOverlayFeature;
+import com.wynntils.utils.MathUtils;
+import com.wynntils.utils.objects.CustomColor;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import com.wynntils.wc.custom.item.properties.type.HighlightProperty;
+import com.wynntils.wc.custom.item.properties.type.TextOverlayProperty;
+import com.wynntils.wc.utils.WynnItemMatchers;
+import java.util.regex.Matcher;
+import net.minecraft.ChatFormatting;
+
+public class PowderTierProperty extends ItemProperty implements HighlightProperty, TextOverlayProperty {
+    private final CustomColor highlightColor;
+
+    private final TextOverlay textOverlay;
+
+    public PowderTierProperty(WynnItemStack item) {
+        super(item);
+
+        // parse color
+        ChatFormatting chatColor =
+                ChatFormatting.getByCode(item.getHoverName().getString().charAt(1));
+        if (chatColor == null) chatColor = ChatFormatting.WHITE;
+
+        highlightColor = CustomColor.fromChatFormatting(chatColor);
+
+        // parse tier
+        Matcher powderMatcher = WynnItemMatchers.powderNameMatcher(item.getHoverName());
+        String powderNumeral = "I";
+        if (powderMatcher.matches()) {
+            powderNumeral = powderMatcher.group(2);
+        }
```

Comment:
What if it doesn't match? Why do we assume TIER 1?

Reply:
the same pattern is used to detect powder items in the first place, so it should be completely impossible for any of it to fail anyway - i didn't choose tier 1 for any particular reason

Reply:
I see, it's fine then I guess.